### PR TITLE
Feat/styling

### PR DESCRIPTION
**Pull request by PhilippWeiss85**
Hi beloved reviewer!
probably the last PR and unfortunately a messy one. What have been done:
- changed and added stylings
- add lottianimation in index.js startpage
- add loading animation (see useStore, gamelist/index.js and form.js)
- add component animations on form, gamedetailsform and lessons)
- add new conditional renderings for image upload in form
- add logo

### CODE REVIEW

File: components/GameCard/GameCard.js [hunk 1]
```diff
@@ -51,23 +53,32 @@ export default function GameCard({
...
           <StyledContent> {place} </StyledContent>
 
-          <CourtHeadline>Court:</CourtHeadline>
-          <StyledContent> {court} </StyledContent>
+          <CourtHeadline>
+            <GiTennisCourt />
+          </CourtHeadline>
+          <StyledContent>{court}</StyledContent>
         </CardList>
+
```
Comment (anhthu253) [inline components/GameCard/GameCard.js:87]: {showMore && ...}

File: components/GameDetailForm/GameDetailForm.js [hunk 2]
```diff
@@ -101,196 +100,202 @@ export default function GameDetails({ id, showMoreDetails, results }) {
...
+            type="number"
+            min="0"
+            max="20"
+            name="secondsetplayertwo"
+            id="secondsetplayertwo"
+            aria-label="Set 2: Score player two"
+            value={secondSetScorePlayerTwo}
+            onChange={(event) => setSecondSetScorePlayerTwo(event.target.value)}
+          />
+        </Fieldsets>
```
Comment (anhthu253) [inline components/GameDetailForm/GameDetailForm.js:196]: as I read the function setValidation() above, I guess players are not allow to set further scores if the first one has not been set yet? I would just display further Fieldsets if the current score has set. Something like set1&&(Fieldsets/) (sori can not write tag here so have to replace with the round bracket). In this way u dont have to write a complicate check up mechanismm for the validation. But for now i guess u can just leave it like that..

File: components/GameCard/GameCard.js (hunk 1 above)
Comment (PhilippWeiss85) [reply components/GameCard/GameCard.js:87]: thanks updated

File: components/GameDetailForm/GameDetailForm.js (hunk 2 above)
Comment (PhilippWeiss85) [reply components/GameDetailForm/GameDetailForm.js:196]: thanks will do that in the future :D

**Review by anhthu253 — APPROVED** [review]

**Pull request merged**